Development environment/Docker.md

```diff
@@ -170,6 +170,10 @@ Remove all images
 
 `docker rmi $(docker images -q)`
 
+Prune system from unused images and containers (use with caution!)
```

Comment:
`use with caution`. Why? What could happen? Just reading it like this makes me not want to run this command ever. (Tho this is just my initial reaction and I have 0 docker experience so take this with a grain of salt). Maybe a bit more detailed caution explanation.

Reply:
I've updated this part. It's tricky as you may remove things you maybe don't want to remove.

Reply:
Much better tho no need to remove the caution. It was just missing some extra info (like the documentation link you added) as to why it's a dangerous command.